.github/workflows/ci.yml

```diff
@@ -73,8 +73,21 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '11'
-      - name: Build the base compiler
+      - name: Build the compiler
         run: ant
       - name: Run Tests
         run: ../bin/pth -ec pthScript-JL pthScript-JL5 pthScript-JL7 pthScript
         working-directory: ./testsjl8
+
+  compile-itself:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - name: Build the compiler
+        run: ant
+      - name: Compile itself
+        run: ./bin/jl8c -j "-Xss40m" -sourcepath src -D temp -assert src/polyglot/ext/jl8/JL8ExtensionInfo.java src/polyglot/main/Main.java
```

Comment:
Cool. How about now invoking the compiled compiler in the same way to see if it generates the same output as the compiler?

Reply:
Addressed in https://github.com/polyglot-compiler/polyglot/pull/58